### DISCUSSION TITLE
Made with AstroChart

### DISCUSSION THREAD

**Discussion by afucher** [AstroDraw/AstroChart#48] [Show and tell]
Please, share here your app, site or github repo that uses AstroChart 🎉 

It helps the library to get visibility and also helps as a motivation to continue to work

**Comment by egorrrrrrrrr** [comment]
I'm doing a simple Astroprocessor service for now, your code it's absolutely amazing, but it not contains some important functionality such setting up As Mc (Ds Ic) points.

**Reply by afucher** [reply]
Hey @egorrrrrrrrr ,

Can you please open an issue for that? Also, if you want to contribute, would be really nice.

**Reply by egorrrrrrrrr** [reply]
Yap, I think I make my service opensource, and try to improve your repo ;)

**Comment by jorjun** [comment]
I made use of AstroChart here: https://greenwichmeantime.com/articles/calendars/birth-chart/

**Comment by zmark9** [comment]
I am using this at rainbowmoon.com - on  pages - find sign and relationship --> link to composite. I have tried to do a synestry/bi-wheel chart  using the transit function - which works but not very user friendly. Not sure if anyone works this project, but it would be great to have one like 
<img width="620" height="632" alt="image" src="https://github.com/user-attachments/assets/3c5291f7-9873-4d7e-b585-7b0ba8d56aa5" />

I may be able to help with programming but svg is a bit out of my wheel house

**Comment by andandio** [comment]
I used it in this project:
https://main.d2syx8lejfpjhs.amplifyapp.com/
<img width="711" height="600" alt="Screenshot 2025-11-23 at 1 10 04 PM" src="https://github.com/user-attachments/assets/9f55ddda-7a51-430e-a98f-f57cd1fd4717" />

**Comment by Gaby-Ridaya** [comment]
Hello, to avoid overlaps in conjunctions, I placed the planet glyphs inside the wheel.
However, in traditional astrology readings, it is still preferable to place the planets outside the wheel.
<img width="660" height="937" alt="Capture d’écran 2026-05-15 190517" src="https://github.com/user-attachments/assets/6bbaa6e0-9636-4ab8-acc9-742bf7cf7f9f" />
<img width="1127" height="794" alt="Capture d’écran 2026-05-15 190707" src="https://github.com/user-attachments/assets/b52f3d31-3697-4872-bd3c-4efe4b7498f2" />

**Comment by hokkar** [comment]
I made a online site https://astrokartographie-analyse.de/
